spec/unit/migrations/helpers_spec.rb

```diff
@@ -0,0 +1,68 @@
+describe Neo4j::Migrations::Helpers do
+  include described_class
+
+  before do
+    clear_model_memory_caches
+    delete_db
+
+    stub_active_node_class('Book') do
+      property :name, constraint: :unique
+      property :author_name, index: :exact
+    end
+
+    Book.create!(name: 'Book1')
+    Book.create!(name: 'Book2')
+    Book.create!(name: 'Book3')
+  end
+
+  describe '#rename_property' do
+    it 'renames a property' do
+      rename_property :Book, :name, :title
+      expect(Book.all(:n).pluck('n.title')).to include('Book1', 'Book2', 'Book3')
+    end
+  end
+
+  describe '#drop_nodes' do
+    it 'drops all nodes given a label' do
+      drop_nodes :Book
+      expect(Book.all.count).to eq(0)
+    end
+  end
+
+  describe '#add_labels' do
+    it 'adds labels to a node' do
+      add_labels :Book, :Item, :Readable
```

Comment:
I think that there should actually be a second argument which is an `Array`.  Looking at this at first I thought you were adding three labels, and then I remembered the method definition.  Like so:

```ruby
add_labels :Book, [:Item, :Readable]
add_label :Book, :Item
```